### DISCUSSION TITLE
build INAV - question

### DISCUSSION THREAD

**Discussion by DanchoDrone** [iNavFlight/inav#8308] [Q&A] [answered]
Hi Everyone
I've pulled inav repo in order to build it with Windows under WSL.
Then I followed the instructions here - https://github.com/iNavFlight/inav/blob/master/docs/development/Building%20in%20Windows%2010%20or%2011%20with%20Linux%20Subsystem.md

The build went good and I had my hex file for the target MAMBAF722_I2C.
After flashing it with the configurator - the configurator was opened in cli and not with the general querstion whether it is a quad copter or a plane...
What am I missing?
Thanks in advance
Dancho

**Comment by DzikuVx** [comment, marked as answer]
You need to build Configurator as well as INAV 6 (master) is not supported by INAV Configurator 5

**Reply by DanchoDrone** [reply]
Thank you for the prompt reply!
I will try it, and have two more questions please as I'm new with this:
1. The build of the configurator is the same process? just clone the configurator repo?
2. Can I clone inav version 5? if yes, please tell me how

**Reply by DanchoDrone** [reply]
@DzikuVx 
I will specify my question - I've looked at the branches and found this one: https://github.com/iNavFlight/inav/tree/release_5.1.0
I wondered if it is possible to clone and work on the latest and stable version from the releases link (https://github.com/iNavFlight/inav/releases).
Thanks
Dancho

**Comment by stronnag** [comment]
as ever http://seyrsnys.myzen.co.uk/inav-configurator-next/

**Reply by DanchoDrone** [reply]
Thanks, from here I can bypass the build of the configurator. But I would like to learn how to do it to specific versions.
Dancho

**Reply by stronnag** [reply]
> Thanks, from here I can bypass the build of the configurator. But I would like to learn how to do it to specific versions. Dancho

The build script is linked on the page ...

**Reply by DanchoDrone** [reply]
@stronnag 
I've seen the build script at the link, but where is the link to a configurator branch to clone (I only see zip files)?
